lib/matplotlib/backends/web_backend/mpl.js

```diff
@@ -394,9 +413,10 @@ mpl.findpos = function(e) {
 mpl.figure.prototype.mouse_event = function(event, name) {
     var canvas_pos = mpl.findpos(event)
 
-    if (this.focus_on_mouseover && name === 'motion_notify')
+    if (name === 'button_press')
```

Comment:
I'm not entirely certain why motion notify is no longer important. Would you mind talking me through it? Does the status bar (with the x and y coordinates) still work with this change?